src/pd/client.rs

```diff
@@ -301,6 +301,10 @@ impl PdClient for RpcClient {
         req.set_keys_read(region_stat.read_bytes);
         req.set_approximate_size(region_stat.approximate_size);
 
+        let now = SystemTime::now();
```

Comment:
do we have a coarse system time? @overvenus

Reply:
Nope, the coarse version should be `clock_gettime(CLOCK_REALTIME_COARSE)`.